vector/src/main/java/im/vector/app/features/onboarding/OnboardingViewModel.kt

```diff
@@ -128,6 +125,9 @@ class OnboardingViewModel @AssistedInject constructor(
         }
     }
 
+    // tchap
```

Comment:
Not related to this change.

I suppose Tchap should return false in this method to not have the server selection in case of error:
```
private fun canEditServerSelectionError(state: OnboardingViewState) =
            (state.onboardingFlow == OnboardingFlow.SignIn && vectorFeatures.isOnboardingCombinedLoginEnabled()) ||
(state.onboardingFlow == OnboardingFlow.SignUp && vectorFeatures.isOnboardingCombinedRegisterEnabled())
```

Reply:
If I set this variable to false, it won't compile saying state is unused.
Do you have a suggestion ?

Reply:
you can add an ignore annotation for the lint issue upon the method